### PR TITLE
Add Korean translation

### PR DESCRIPTION
**Pull request by memnoth**
1st version of Korean translation for new_layout branch.

### CODE REVIEW

File: youtube_dl_gui/locale/ko_KR/LC_MESSAGES/youtube_dl_gui.po [hunk 1]
```diff
@@ -0,0 +1,582 @@
+# Youtube-dlG localization file.
+# FIRST AUTHOR: Sotiris Papadopoulos <ytubedlg@gmail.com>, 2015.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: youtube-dlg 0.3.8\n"
+"POT-Creation-Date: 2016-12-17 17:07+EET\n"
+"PO-Revision-Date: 2016-12-17 17:08+EET\n"
```
Comment (MrS0m30n3) [inline youtube_dl_gui/locale/ko_KR/LC_MESSAGES/youtube_dl_gui.po]: Please update the revision date

File: youtube_dl_gui/locale/ko_KR/LC_MESSAGES/youtube_dl_gui.po [hunk 2]
```diff
@@ -0,0 +1,582 @@
...
+msgid "Turkish"
+msgstr "Turkish"
+
+#: optionsframe.py:567
+msgid "None"
+msgstr "없음"
+
+#: optionsframe.py:568
+msgid "Automatic subtitles (YOUTUBE ONLY)"
+msgstr "Automatic subtitles (유튜브 전용)"
```
Comment (MrS0m30n3) [inline youtube_dl_gui/locale/ko_KR/LC_MESSAGES/youtube_dl_gui.po]: You should translate this one

File: youtube_dl_gui/locale/ko_KR/LC_MESSAGES/youtube_dl_gui.po [hunk 3]
```diff
@@ -0,0 +1,582 @@
...
+
+#: optionsframe.py:569
+msgid "All available subtitles"
+msgstr "사용가능한 모든 자막"
+
+#: optionsframe.py:570
+msgid "Subtitles by language"
+msgstr "Subtitles by language"
```
Comment (MrS0m30n3) [inline youtube_dl_gui/locale/ko_KR/LC_MESSAGES/youtube_dl_gui.po]: And this one

**Comment by memnoth** [comment]
Edit revision date.
Translate everything.

**Comment by MrS0m30n3** [comment]
Thanks for your contribution!

**Pull request merged**